Sources/Testing/ExitTests/SpawnProcess.swift

```diff
@@ -143,12 +143,14 @@ func spawnExecutable(
 #if SWT_TARGET_OS_APPLE
       // Close all other file descriptors open in the parent.
       flags |= CShort(POSIX_SPAWN_CLOEXEC_DEFAULT)
-#elseif os(Linux) || os(FreeBSD)
+#elseif os(Linux)
       // This platform doesn't have POSIX_SPAWN_CLOEXEC_DEFAULT, but we can at
       // least close all file descriptors higher than the highest inherited one.
       // We are assuming here that the caller didn't set FD_CLOEXEC on any of
       // these file descriptors.
       _ = swt_posix_spawn_file_actions_addclosefrom_np(fileActions, highestFD + 1)
+#elseif os(FreeBSD)
+      _ = posix_spawn_file_actions_addclosefrom_np(fileActions, highestFD + 1)
```

Comment:
Can you explain why this change was needed?

Reply:
FreeBSD do not use Glibc, and all non-EOL (13.1+) FreeBSD libc [supports `posix_spawn_file_actions_addclosefrom_np`](https://man.freebsd.org/cgi/man.cgi?query=posix_spawn_file_actions_addclosefrom_np&apropos=0&sektion=0&manpath=FreeBSD+14.2-RELEASE+and+Ports&arch=default&format=html).

`swt_posix_spawn_file_actions_addclosefrom_np` on the other hand is defined in `Stubs.cpp` to guard the actual implementation behind `#if defined(__GLIBC__)` and `__GLIBC_PREREQ(2, 34)`, such that Linux host with lower glibc version can still works (and becomes no-op). And changing `Stubs.cpp` will end up with a less trivial diff.

Reply:
Of course. :) Mind adding a brief comment above the function call so folks reading the code know why it's different?

Reply:
ofc! done!